diesel/src/macros/mod.rs

```diff
@@ -177,6 +177,23 @@ macro_rules! __diesel_column {
 /// # }
 /// ```
 ///
+/// If you are using types that aren't from Diesel's core types, you can specify
+/// which types to import. Note that the path given has to be an absolute path
+/// relative to the crate root. You cannot use `self` or `super`.
+///
+/// ```ignore
+/// #[macro_use] extern crate diesel;
+/// extern crate diesel_full_text_search;
+///
+/// table! {
+///     posts using types from diesel::types, diesel_full_text_search::types {
```

Comment:
I think the `use` syntax (https://github.com/diesel-rs/diesel/pull/779#pullrequestreview-25265816) is nicer. But in any case, it'd be great to document the full syntax; at least with an example that uses everything.